functions.js

```diff
@@ -0,0 +1,37 @@
+const symbols = require('./symbols');
+
+module.exports = {
```

Comment:
This type of yucky `model.exports = {` can be avoided using ECMAScript exports
and imports, which are much nicer. nodejs does have support for these in later
versions (I believe) but I might be wrong.

Reply:
Yeah I totally would, but idk it was giving me errors when I was trying to export stuff so I'm just sticking with what they use in the docs for now